### DISCUSSION TITLE
Support for NODEJS Runtime

### DISCUSSION THREAD

**Discussion by ScreamZ** [motiondivision/motion#2886] [Q&A]
Hi,

Do you plan to support NodeJS runtime to animate value on server application. I used to play with promotion to animate values for DMX (Light controller) and also lights on IoT device.

Can i safely move to motion ?

Thanks

**Comment by mattgperry** [comment]
I believe so, all our tests run on Node.js - why not try it and let us know?

**Reply by ScreamZ** [reply]
Sure I'll sir! Thanks for reactivity :) I'll update this topic accordingly.
